### PR TITLE
Version 1.1.0

### PR DESCRIPTION
**Pull request by RyanBrushett**
**What are you trying to accomplish with this PR?**

Cut version 1.1.0

**What could go wrong?**

Did I catch every change since 1.0.0?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## next
 
+## 1.1.0
+
 *Bug Fixes*
 - Fix a bug causing secret generation from ejson to fail when decryption succeeded but a warning was also emitted. [#647](https://github.com/Shopify/krane/pull/647)
```
Comment (RyanBrushett) [inline CHANGELOG.md]: @timothysmith0609 is this the changelog entry you mean?

Comment (timothysmith0609) [reply CHANGELOG.md]: :facepalm: I don’t appear to be awake today

Comment (timothysmith0609) [reply CHANGELOG.md]: :facepalm: I don’t appear to be awake today

Comment (RyanBrushett) [reply CHANGELOG.md]: Haha it's OK <3

**Pull request merged**